src/meson.build

```
@@ -128,11 +134,12 @@ gtkwave_include_directories = [
     'cocoa',
 ]
 
-executable(
+exe = executable(
```

The variable name should be more specific because there are multiple executables.
```suggestion
gtkwave_exe = executable(
```